consumer/consumer.go

```diff
@@ -0,0 +1,192 @@
+package consumer
+
+import (
+	"fmt"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/Shopify/sarama"
+	cluster "github.com/bsm/sarama-cluster"
+	"github.com/heetch/felice/consumer/handler"
+	"github.com/heetch/felice/message"
+	"github.com/pkg/errors"
+)
+
+// Consumer is a Kafka consumer.
```

Comment:
:tulip: this isn't a great comment, but I'll pick this up in the documentation work.